### DISCUSSION TITLE
[ FF7 ] 7th Heaven 2.3 Crash

### DISCUSSION THREAD

**Discussion by KyleFFVII** [julianxhokaxhiu/FFNx#208] [General]
After using 7th Heaven 2.2 I continued to come to the same crash where it would fail to launch with any mods. This time around I get to the "new game" "continue" screen, with mods loaded. The moment I select new game though, the game crashes with the attached crash log.

[Compressed FF7 Crash.zip](https://github.com/julianxhokaxhiu/FFNx/files/6077836/Compressed.FF7.Crash.zip)

**Comment by eve-atum** [comment]
Disable the Qhimm-Animations mod
